chapters/4-acquisition.tex

```diff
@@ -331,29 +350,29 @@ \subsection{Developing a data collection instrument}
 
 
 At this point, it is useful to do a \textbf{survey pilot}\sidenote{
- \url{https://dimewiki.worldbank.org/Piloting\_Survey\_Content}}\index{survey pilot}
+  More details on how to structure survey pilots,
+	the different components of comprehensive survey pilot and
+	link to many other resources can be found on the DIME Wiki:
+ \url{https://dimewiki.worldbank.org/Survey\_Pilot}}\index{survey pilot}
 of the questionnaire.
-A comprehensive survey pilot is a critical stage in survey design.\sidenote{
-	\url{https://dimewiki.worldbank.org/Survey\_Pilot}}
+A comprehensive survey pilot is a critical stage in survey design.
 The pilot must be done out-of-sample,
 but in a context as similar as possible to the study sample.
```

Comment:
add link to https://dimewiki.worldbank.org/wiki/Survey_Pilot_Participants here

Reply:
done in 04a6a926ec8986149a58e7f5e0f4dc8848a1dc3a